src/main/webapp/instructorHelp.html

```diff
@@ -7461,7 +7568,7 @@ <h5 class="col-sm-2">
                                             Team 3
                                         </td>
                                         <td>
-                                            
+
```

Comment:
empty line shouldn't be indented, right ? I am not sure about this convention. Will revert back if you confirm.